### DISCUSSION TITLE
Obvious method to save UI space.

### DISCUSSION THREAD

**Discussion by J1C0** [rxhanson/RectanglePro-Community#443] [General]
I used the Mac 3rd party window manager Magnet.
It does 1 excellent thing far better than Rectangle does.  It conserves space in the popup menu, by clustering snapping options together.  For example, to divide up a 32:9 screen into quarters, rather than having separate 4 buttons like rectangle does:
![Screenshot 2024-02-05 at 6 13 40 PM](https://github.com/rxhanson/RectanglePro-Community/assets/33563035/05954a34-d90f-4dc7-aca3-c646037935b7)

Magnet instead clusters all 4 together like this:
![image](https://github.com/rxhanson/RectanglePro-Community/assets/33563035/bbe2a62a-a37f-4aaa-b2d7-9f18917d96af)

By conserving space this way, you can fit way more options on a given size of popup menu:
![Screenshot 2024-02-05 at 6 11 39 PM](https://github.com/rxhanson/RectanglePro-Community/assets/33563035/6bd7170a-dece-46d7-a2ad-77c4ff4a74ff)

This is superior design.

**Comment by rxhanson** [comment]
Thanks for the feedback. I'll keep this in mind when I'm working on snap panel updates.
